### ISSUE TITLE
VS 2015 designer stops working whenever a data bound object is a CSLA object

### ISSUE THREAD

**Issue by Nilzor** [MarimerLLC/csla#661]
Not sure if this is should be addressed Microsoft or MarimerLLC but the fact is that whenever a class extends a CSLA BusinessBase object using CSLA-UWP 4.6.401 and that class is referenced in the designer as DataContext, the designer stops working. 

Example project: (Check out branch `designer-issue`, switch between `PlainClass` and `CslaClass`, rebuild and observe): https://github.com/Nilzor/csla-uwp-ntcfail/tree/designer-issue

![image](https://cloud.githubusercontent.com/assets/990654/20040411/3821cfac-a457-11e6-8a85-5cd459a8989d.png)

Tested on Windows 10 build 10586, VS 2015 
Microsoft Visual Studio Community 2015
Version 14.0.25425.01 Update 3

**Comment by Nilzor** [comment]
Bump. Still an issue, and I'm surprised no one else has commented. Is that because I'm the only one who's using CSLA-UWP, or becuase I'm the only one using the designer. From my point of view this is a pretty serious issue:

**The Visual Studio designer does not work with CSLA-UWP**

It may or may not be related to [another exception](https://github.com/MarimerLLC/csla/issues/535#issuecomment-275916700) I see from time to time in the designer:

```
TypeLoadException: Type 'Csla.Core.BusinessBase' in assembly 'Csla, Version=4.6.500.0, Culture=neutral, PublicKeyToken=93be5fdc093e4c30' has method 'OnDeserializedHandler' with an incorrect signature for the serialization attribute that it is decorated with.
```

**Comment by rockfordlhotka** [comment]
I can't reproduce this issue in VS 2017 with CSLA 4.7.200, so I'm going to close the issue.

That problem with the `OnDeserializedHandler` signature was addressed (I think) in 4.7.100 actually - I have some memory of solving it, probably as part of implementing support for netstandard 2.0.
